src/download.ts

```diff
@@ -180,7 +180,7 @@ export async function downloadFileHns(
  * Constructs the full URL for the given skylink.
  *
  * @param this - SkynetClient
- * @param skylinkUrl - Skylink string. See `downloadFile`.
+ * @param skylinkUrl - Base64 skylink, or a valid URL that contains a skylink. See `downloadFile`.
```

Comment:
does it also accept base 32 skylink? I think it should right?

Reply:
Nope. `parseSkylink` only detects base32 if the `fromSubdomain` option is set and the input is a URL. That method probably needs an overhaul to automatically detect the skylink type.

Related: https://github.com/SkynetLabs/skynet-js/issues/238

Reply:
More directly related: https://github.com/SkynetLabs/skynet-js/issues/237

Just haven't had time to make this change I guess.